resources/metadata/abapEnvironmentRunAUnitTest.yaml

```diff
@@ -128,6 +128,14 @@ spec:
           - STEPS
         mandatory: false
         default: "AUnitResults.xml"
+      - name: generateHTML
```

Comment:
As we have 2 generateHTML flags, we could add the GENERAL scope to both steps.

Reply:
Done